### PR TITLE
ADF-3953 and ADF-3963

### PR DESCRIPTION
**Pull request by gmandakini**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-3963
https://issues.alfresco.com/jira/browse/ADF-3953

### CODE REVIEW

File: e2e/core/viewer/viewer-component.e2e.ts [hunk 1]
```diff
@@ -203,7 +203,7 @@ xdescribe('Viewer', () => {
 
     });
 
-    describe('PowerPoint Folder Uploaded', () => {
+    xdescribe('PowerPoint Folder Uploaded', () => {
```
Comment (eromano) [inline e2e/core/viewer/viewer-component.e2e.ts]: please do not exclude any e2e

File: e2e/core/viewer/viewer-component.e2e.ts [hunk 2]
```diff
@@ -430,7 +430,7 @@ xdescribe('Viewer', () => {
             viewerPage.checkImgContainerIsDisplayed();
         });
 
-        it('[C260106] Should be able to open a Word file shared via API', () => {
+        xit('[C260106] Should be able to open a Word file shared via API', () => {
```
Comment (eromano) [inline e2e/core/viewer/viewer-component.e2e.ts]: please do not exclude any e2e

File: e2e/core/viewer/viewer-component.e2e.ts (hunk 1 above)
Comment (gmandakini) [reply e2e/core/viewer/viewer-component.e2e.ts]: These have been excluded, because there are related bugs - ADF-3953 and ADF-3963

Comment (eromano) [reply e2e/core/viewer/viewer-component.e2e.ts]: In general, we don't comment out a test. The options are:

1. There is a regression we fix the regression
2. The use case is not valid anymore we delete the test
3. The test has a problem we fix the test

Comment (gmandakini) [reply e2e/core/viewer/viewer-component.e2e.ts]: I think, the use case is valid, the Test seems to be fine as well. the 2 Jira's need fixing ADF-3953 and ADF-3963

**Comment by eromano** [comment]
I am closing this PR

**Pull request closed without merging**